app/views/posts/show.html.erb

```diff
@@ -8,4 +8,25 @@
     </div>
     <p><%= @post.text %></p>
   </div>
+
+  <div class="comments-list">
+    <% comments = Comment.where(post_id: @post.id) %>
+    <% comments.each do |comment| %>
+    <span><% user = User.find_by(id: comment.author_id).name %></span>
+    <span><%= user %>: <%= comment.text %> </span>
+    <% end %>
+  </div>
+
+  <div class="add-comment">
+  <%= form_with model: @comment, url: user_post_comments_path(@user.id, @post.id) do |form| %>
+    <%= form.text_field :text %>
+    <%= form.submit "Comment" %>
+    <% end %>
+  </div>
+
+  <div>
+    <%= form_with model: @like, url: user_post_likes_path(@user.id, @post.id) do |form| %>
+    <%= form.submit "Like" %>
+    <% end %>
+  </div>
```

Comment:
- [ ] I could not view the form or test it out because this project **_has no migration_** files. Please include the migration files because they define tables and relationships between the models in the database. We can't test the codes you wrote for this milestone without them. 👍 😃 

![image](https://user-images.githubusercontent.com/77106962/229570929-43d21c1e-8935-4825-a58d-37b43896e13d.png)